midend/local_copyprop.cpp

```diff
@@ -228,6 +260,26 @@ IR::AssignmentStatement *DoLocalCopyPropagation::postorder(IR::AssignmentStateme
     return as;
 }
 
+IR::IfStatement *DoLocalCopyPropagation::postorder(IR::IfStatement *s) {
```

Comment:
I would expect the immediately following call to `ElimDead` to do this?  Are there cases where it does not?